app/Console/Commands/CacheLeaderboard.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Enum\CacheKey;
+use App\Http\Controllers\Frontend\LeaderboardController;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Facades\DB;
+
+class CacheLeaderboard extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'trwl:leaderboard:cache';
```

Comment:
I would switch the keys, so there is a better scheme, if we add another caching tasks later.

```suggestion
    protected $signature = 'trwl:cache:leaderboard';
```